docs/README.md

```diff
@@ -38,6 +38,8 @@ Once in the app's settings page, define the following settings:
 
 - **Default all review accordions to open**: Checking this box will cause all review accordions on the product page to default to open with review text limited to 3 lines. Reviews with more than 3 lines of text will be truncated with an ellipsis and a "Show More" link that can be used to display the whole review text.
 
+- **Number of open review accordions**: Select the number of review accordions on the product page to default to open. If the default all review accordions to open is active, this option is ignored.
```

Comment:
I think this part is a bit confusing right now.

What are your thoughts? @guerreirobeatriz

Reply:
I don't understand either! 

Also, I don't find this in the app's settings section when using the Admin. @SDBowen Could you help me with this? :)

Reply:
Hmm, [does this direct link work](https://expandedreviews--sandboxusdev.myvtex.com/admin/apps/vtex.reviews-and-ratings@2.0.3/setup/)?

I agree it is not too clear yet. Expanding on it a bit:

> The specified number of review accordions on the product page will default to open, displaying all the review text. This option provides an alternative to defaulting all review accordions to open and allows a specific number of reviews to be open by default.

Reply:
@guerreirobeatriz You can try this link too: https://reviews--redoxx.myvtex.com/admin

Reply:
@guerreirobeatriz To explain further: on the reviews page, the app displays a paginated list of reviews with 10 reviews per page. Each review is inside a collapsible accordion. The app already has an app setting that determines whether the accordions start closed or open, but this setting applies to all of them; i.e. either all of the reviews start expanded, or all of them start collapsed. This new setting allows the client to set a specific number of reviews to be expanded when the page is loaded. They could show 3 expanded and the rest closed, or 5, or whatever. This was a feature request that our team received.